server/custom_vm_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package server
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReplaceWhitelistedSubnets(t *testing.T) {
```

Comment:
probably a general function to replace into json string shall be used in many places